2-structured-data/package.json

```diff
@@ -40,7 +40,6 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
-    "gcloud": "~0.37.0",
     "@google-cloud/datastore": "^0.1.1",
```

Comment:
The latest version of `@google-cloud/datastore` is `0.4.0`. Are you wanting to upgrade in this PR?